models/School/SchoolParser.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace app\models\School;
+
+use \UnexpectedValueException;
+use app\models\School\SchoolCollection;
+
+class SchoolParser
+{
+    const HEAD_COUNT_KEY_PREFIX = 'HC_';
```

Comment:
Set this as a constant prefix, in case one day they switch out the keyname to something else, this is easily identifiable.